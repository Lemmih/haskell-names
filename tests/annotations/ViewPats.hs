{-# LANGUAGE ViewPatterns #-}
module ViewPats where

f (y -> x) z@(x -> y) = do
  x
  y
  z
