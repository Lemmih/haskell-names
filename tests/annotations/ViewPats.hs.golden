y        at  4:4 is not in scope
x        at 4:15 is a local value defined at 4:9
x        at  5:3 is a local value defined at 4:9
y        at  6:3 is a local value defined at 4:20
z        at  7:3 is a local value defined at 4:12
